doc/nrf/libraries/networking/download_client.rst

```diff
@@ -8,37 +8,53 @@ Download client
    :depth: 2
 
 The download client library can be used to download files from an HTTP or a CoAP server.
+
+Overview
+********
+
 The download is carried out in a separate thread, and the application receives events such as :c:enumerator:`DOWNLOAD_CLIENT_EVT_FRAGMENT` that contain the data fragments as the download progresses.
-The fragment size can be configured independently for HTTP and CoAP (block-wise transfer) using the :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_HTTP_FRAG_SIZE` and the :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_COAP_BLOCK_SIZE` options, respectively.
+The fragment size can be configured independently for HTTP and CoAP (block-wise transfer).
 When the download completes, the library sends the :c:enumerator:`DOWNLOAD_CLIENT_EVT_DONE` event to the application.
 
 Protocols
-*********
+=========
 
 The library supports HTTP, HTTPS (TLS 1.2), CoAP, and CoAPS (DTLS 1.2) over IPv4 and IPv6.
 
+.. _download_client_https:
 
 HTTP and HTTPS (TLS 1.2)
 ========================
```

Comment:
@greg-fer Shouldn't this be in h4? ( -------- )
It should be contained under the protocols section (currently in h3).